addons/website_slides/models/slide_channel.py

```diff
@@ -801,15 +801,33 @@ def toggle_active(self):
     def message_post(self, *, parent_id=False, subtype_id=False, **kwargs):
         """ Temporary workaround to avoid spam. If someone replies on a channel
         through the 'Presentation Published' email, it should be considered as a
-        note as we don't want all channel followers to be notified of this answer. """
+        note as we don't want all channel followers to be notified of this answer.
+        Also make sure that only one review can be posted per course."""
         self.ensure_one()
         if kwargs.get('message_type') == 'comment' and not self.can_review:
             raise AccessError(_('Not enough karma to review'))
         if parent_id:
             parent_message = self.env['mail.message'].sudo().browse(parent_id)
             if parent_message.subtype_id and parent_message.subtype_id == self.env.ref('website_slides.mt_channel_slide_published'):
                 subtype_id = self.env.ref('mail.mt_note').id
-        return super(Channel, self).message_post(parent_id=parent_id, subtype_id=subtype_id, **kwargs)
+        message = super().message_post(parent_id=parent_id, subtype_id=subtype_id, **kwargs)
+        if message.subtype_id == self.env.ref("mail.mt_comment"):
+            if (
+                self.env["mail.message"].search_count(
+                    [
+                        ("res_id", "=", self.id),
+                        ("author_id", "=", message.author_id.id),
+                        ("model", "=", "slide.channel"),
+                        ("subtype_id", "=", self.env.ref("mail.mt_comment").id),
+                    ],
+                    limit=2,
+                )
+                > 1
```

Comment:
Please, use your whole screen

Reply:
Same as the other comment. What is the suggestion for formatter rather than black -l 100?

Reply:
I personally don't care but if you must you can always split complex lines with well-chosen variables.
Example here:
```suggestion
                domain = [
                    ("res_id", "=", self.id),
                    ("author_id", "=", message.author_id.id),
                    ("model", "=", "slide.channel"),
                    ("subtype_id", "=", self.env.ref("mail.mt_comment").id),
                ]
                if (self.env["mail.message"].search_count(domain, limit=2) > 1)
```

Reply:
Do it manually, you will always obtain better results compared to automated tools. Official python odoo recommendation is: rulers at 80, 100 and 120 and use your best writing practice to somehow be between 80 and 120 while easing readability. Anything else is just some nazis trying to lure you (beware those are nasty).